tests/layer_tests/pytorch_tests/test_norm.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, p, dim, keepdim) -> None:
                 self.keepdim = keepdim
 
             def forward(self, input_data):
-                return torch.norm(input_data, self.p, self.dim, self.keepdim)
+                return torch._VF.norm(input_data, self.p, self.dim, self.keepdim)
```

Comment:
Not sure about this change, norm works on my system without it

Reply:
it will be removed in torch 2.0.0, so as we do not know when it will be merged to openvino/master, better to be ready to support it

Reply:
Okay